docs/running-jobs/job-and-queue-scheduling.md

```diff
@@ -1,55 +1,128 @@
+`ngpus`: The number of allocable GPUs on the vnode.  For an Nvidia A100, this could be one, however, if we enable *Multi Instance GPU (MIG)* mode and use cgroups it could be as high as 7.
 
-### The basics
-If you are an existing ALCF user and are familiar with Cobalt, you will find the PBS commands very similar though the options to qsub are quite different.  Here are the "Big Four" commands you will use:
+## <a name="Quick-Start"></a>Quick Start
+If you are an ALCF user and are familiar with Cobalt, you will find the PBS commands very similar though the options to qsub are quite different.  Here are the "Big Four" commands you will use:
 
-1. `qsub` - request resources (compute nodes) to run your job and start your script/executable on the head node.
-2. `qstat` - check on the status of your request
+1. `qsub` - request resources (generally compute nodes) to run your job and start your script/executable on the head node.  Here is the minimal qsub allowed at the ALCF:
+    * `qsub -A <project> -l select=<# of nodes>,walltime=HH:MM:SS,filesystems=fs1:fs2 <your job script>`
+    * The `-A`, `walltime`, and `filesystems` are mandatory.  You will receive errors if they are not specified.
+    * We automatically add `-k doe` for you, but specifying it explicitly probably wouldn't hurt (streams your output back rather than spooling it and copying it back at the end of the job)
+    * It is highly likely you will also want to add `-l place=scatter` so that each of your chunks (`<# of nodes>`) gets its own vnode. 
+    * If you want to run an executable rather than a script replace `<your jobs script>` in the example above with `-- <your executable>` (that is dash dash)
+    * PBS Documentation: Users Guide, Chapter 2, page UG-11 and Reference Guide Chapter 2, section 2.57, page RG-216
+2. `qstat` - check on the status of your jobs or queues
+    * Try these variations and see which you like best: `qstat`, `qstat -was`, `qstat -was1`, `qstat -wan`, `qstat -wan1`; Add `-x` to see jobs that have completed.  We keep two weeks of history.
+    * `qstat -Q` will list all the queues in case you forget.
+    * PBS Documentation: Users Guide Sec. 10.2, page UG-175; Reference Guide Sec. 2.55, page RG-200 
 3. `qalter` - update your request for resources
-4. `qdel` - cancel an uneeded request for resources
+    * Just like qsub, just add a jobid at the end.  Only works before the job starts;
+    * If you want to change the walltime to 30 minutes: `qalter -l walltime=30:00:00 <jobid>`
+    * PBS Documentation: Users Guide Sec. 9.2, page UG-168; Reference Guide Sec. 2.40, page RG-130
+4. `qdel` - cancel a job that you don't need; This will also kill a running job
+    * `qdel <jobid>`
+    * PBS Documentation: Users Guide Sec. 9.3, page UG-170; Reference Guide Sec. 2.41, page RG-143
+
+**Note: The page numbers in the PBS guides are unique.  If you search for the specified page number it will take you directly to the relevant page.**
```

Comment:
We will need to check these page numbers and the like as we update PBS versions as changes to doc should shift them.